### PR TITLE
add contact us action to overflow menu

### PR DESCRIPTION
**Pull request by kriticalflare**

### CODE REVIEW

**Comment by prabhuomkar** [comment]
@kriticalflare resolve conflicts

**Comment by kriticalflare** [comment]
@prabhuomkar should be fine now

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -2,9 +2,8 @@
     <string name="app_name">TorchExpo</string>
     <string name="app_description">Collection of machine learning experiments using PyTorch Android API</string>
     <string name="app_version">v%1$s</string>
-
-    <string name="options_menu_about">About</string>
-    <string name="options_menu_github">GitHub</string>
-
+    <string name="options_menu_github">Github</string>
+    <string name="options_menu_contact_us">Contact us</string>
```
Comment (prabhuomkar) [inline app/src/main/res/values/strings.xml:6]: I don't think this will work.
You missed "options_menu_about" and updated "options_menu_github"

Comment (prabhuomkar) [reply app/src/main/res/values/strings.xml:6]: Please test locally after fixing merge conflict :)

**Pull request merged**